### PR TITLE
Add nav-link-active styles

### PR DESCRIPTION
**Pull request by jonathanchaffer**

### CODE REVIEW

File: client/src/components/layout/Navbar.jsx [hunk 1]
```diff
@@ -23,15 +22,21 @@ const Navbar = withRouter(({ history }) => {
...
-                            alt=""
-                            style={{ height: "60px", marginRight: ".5rem" }}
-                        />
-                    </div>
-                )}
+                <div
+                    className="navbar-icon"
+                    style={{
+                        opacity: isHomePage ? 0 : 1,
+                    }}
```
Comment (zachsnoek) [inline client/src/components/layout/Navbar.jsx:30]: you're the man!!!

**Comment by zachsnoek** [comment]
Looking good! Can you add the dark background around the text when the active class is applied, like when you hover over a link?

Screen Shot 2020-11-29 at 9 09 05 AM

**Comment by zachsnoek** [comment]
I think we lost the gray color on the "Zach Snoek" text hover; now it's blue but it used to be this color:

Screen Shot 2020-11-29 at 9 35 33 AM

**Comment by jonathanchaffer** [comment]
> I think we lost the gray color on the "Zach Snoek" text hover; now it's blue but it used to be this color:
> 
> Screen Shot 2020-11-29 at 9 35 33 AM

Good catch.

**Comment by zachsnoek** [comment]
I'm not seeing the gray background in the active `NavLink` class anymore on the [PR deploy preview](https://deploy-preview-6--pedantic-lichterman-c7bee0.netlify.app/portfolio)...

**Pull request merged**